src/python/ensembl/database/unittestdb.py

```diff
@@ -0,0 +1,174 @@
+# .. See the NOTICE file distributed with this work for additional information
+#    regarding copyright ownership.
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#        http://www.apache.org/licenses/LICENSE-2.0
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""Unit testing database handler.
+
+This module provides the main class to create and drop unit testing databases, populating them from
+preexisting dumps (if supplied).
+
+Typical usage example::
+
+    db = UnitTestDB('mysql://ensro@mysql-server:4242/', 'path/to/dumps', 'my_db')
```

Comment:
Can you add the necessary import statement ? And change this to `ensembl.database.UnitTestDB` or whatever namespace the class is in